compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/CanvasLayersComposeScene.skiko.kt

```diff
@@ -253,6 +254,9 @@ private class CanvasLayersComposeSceneImpl(
     override fun processKeyEvent(keyEvent: KeyEvent): Boolean =
         focusedLayer?.onKeyEvent(keyEvent) ?: mainOwner.onKeyEvent(keyEvent)
 
+    override fun processRotaryScrollEvent(event: RotaryScrollEvent): Boolean =
+        mainOwner.focusOwner.dispatchRotaryEvent(event)
```

Comment:
It should be something like
```suggestion
        focusedLayer?.focusOwner.dispatchRotaryEvent(keyEvent) ?: mainOwner.focusOwner.dispatchRotaryEvent(event)
```

To properly forward it to Popup/Dialog

Reply:
Also, it's probably better to add `RootNodeOwner.onRotaryEvent` to avoid `focusOwner` chaining here

Reply:
Done! What about this?